deploy/platform/redhat/redhat_docker_build.sh

```diff
@@ -23,6 +23,10 @@ do_createrepo() {
   fi
   : && createrepo -q $update_args -o ${tmpdir} ${repodir}/${FLAVOR}/RPMS
   checkstatus abort "Failure: Problem creating rpm repository in ${repodir}!" $?
+  rm -f ${tmpdir}/repodata/repomd.xml.asc
+  export GNUPGHOME=/sign_keys/.gnupg
```

Comment:
Note that when Docker runs, it maps the `--keys=<dir>` to `/sign_keys` in the workspace.   The hard-coded `/sign_keys` occurs in 3 other locations in this file.   If we replace `/sign_keys` with an environment variable, then the change should be made to all occurrences.